.github/workflows/release-package.yml

```diff
@@ -0,0 +1,61 @@
+name: Build RPM and DEB for latest Release
+
+on:
+  release:
+    types: [created] # Only run on latest manual release
+  workflow_dispatch: # Enable manual runs to edit existing releases
+
+jobs:
+  build:
+    name: Build DEB and RPM
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v4
+
+    - name: Set up Rust
+      uses: actions-rust-lang/setup-rust-toolchain@v1
+
+    - name: Install Tools
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y rpm build-essential
+        cargo install cargo-deb
+        cargo install cargo-rpm
+
+    - name: Build .deb package
+      run: cargo deb
+
+    - name: Build .rpm package
+      run: |
+        cargo rpm build
+        mkdir -p target/release
+        cp target/release/rpmbuild/RPMS/*/*.rpm target/release/
+
+    - name: Get latest release
+      id: get_tag
+      uses: actions/github-script@v7
+      with:
+        script: |
+          if(github.event_name === 'release') {
+            return github.event.release.tag_name;
+          } else {
+            // Manual dispatch: Fetch latest release tag
+            const latestRelease = await github.rest.repos.getLatestRelease({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+            });
+            return latestRelease.data.tag_name;
+          }
```

Comment:
I am not a fan of this code. In its current state, one could accidentally run the workflow for the `main` branch, and then this script would re-release a random commit with the last name of the Git tag. I would like to see that this (or a separate step) guards against such errors.

Reply:
You're right, i added verficiation so the action fails if the current commit does not match up with the latest tag so we don't build on a random commit.